packages/engine-formula/src/functions/text/find/__test__/index.spec.ts

```diff
@@ -0,0 +1,151 @@
+/**
```

Comment:
I suggest that the newly created feature branch should not carry the code of other feature branches, which will make the original feature branch useless. You can close the PR submitted by the source branch (such as https://github.com/dream-num/univer/pull/2547), or create a new branch from dev to develop the new function LEFT function